### DISCUSSION TITLE
prompt before actions on a bk-table component

### DISCUSSION THREAD

**Discussion by davidgarra** [mia-platform/community#474] [Q&A] [answered]
### What product are you having troubles with?

Backoffice

### Which is the version?

back-kit@1.4.7, microlc-backend@0.8.1, microlc-frontend@0.9.0

### Description

On a `bk-table`, I have an action and I would like to prompt the user to enter a code (which will be sent in the POST request). I'd like to use a simple prompt-like component for this. Is this possible?

Just to better explain, I'm thinking of something like this (see `requirePrompt`):
```json
{
  "label": {
    "it": "it",
    "en": "en"
  },
  "kind": "httpPost",
  "content": "/my-endpoint",
  "requirePrompt": {
    "title": "Prompt",
    "content": "Insert code"
  }
}
```

Is this possible?

**Comment by clabene** [comment, marked as answer]
Hi @davidgarra, thanks for reaching out! This cannot be achieved as of now, but our road-map includes features that will allow to reach your goal.
We will keep you posted on the progress!
You can check out what is included in each release [here](https://docs.mia-platform.eu/docs/release-notes/versions)
